src/view/components/Login/Login.svelte

```diff
@@ -23,7 +29,27 @@
     if (await user.get()) {
       userLoggedIn = true;
     }
+    addToIDB();
   });
+
+  const addToIDB = () => {
```

Comment:
This looks like everything can be done in one function in three steps

1. Listen for the event
2. If the event is the firebase user (you re not currently checking this) and it comes from ticketland dapp (this might not be needed) then call this logic
3. set `userLoggedIn` to true